server/badge/badge.go

```diff
@@ -107,7 +107,7 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 	//Sample url: http://localhost:8080/api/badge?name=123&revision=true
-	if _, ok := r.URL.Query()["revision"]; ok && enabled {
+	if revisionParam, ok := r.URL.Query()["revision"]; ok && enabled && revisionParam[0] == "true" {
```

Comment:
Could we make the comparison case-insensitive? It's a breaking API change, but I think minor enough to be acceptable in a patch release. But I'd like to minimize potential breakage.

Reply:
good one, applied it with `EqualFold`